subsys/suit/memory_layout/Kconfig

```diff
@@ -15,7 +15,7 @@ config SUIT_MEMORY_LAYOUT_EXTMEM_ADDRESS_RANGE_START
 
 config SUIT_MEMORY_LAYOUT_EXTMEM_ADDRESS_RANGE_SIZE
 	hex
-	default 0x40000000 if SOC_NRF54H20
+	default 0x40000000 if SOC_SERIES_NRF54HX
 	default 0x8000000 if SOC_NRF52840
 	default 0x0 if BOARD_NATIVE_POSIX
 	help
```

Comment:
and this, this is a hardware property which should come from dts

Reply:
Same here - I'll add a note in the task describing flash_companion image migration.